src/styles/modules/_toast.scss

```diff
@@ -0,0 +1,51 @@
+.dc-toast {
+    position: fixed;
+    top: 0;
+    right: 0;
+    bottom: 0;
+    left: 0;
+    transition: all linear .3s;
+    z-index: 9000;
+
+    &.dc-toast--hidden {
+        // For animating the toast entry/leave from the screen
+        top: -36px;
+    }
```

Comment:
Good catch. I think a toast should have at max one line so I will force a max-height and set the correct overflow mode.